modules/distribution/src/main/resources/log4j.properties

```diff
@@ -92,7 +92,7 @@ log4j.logger.trace.messages=TRACE,CARBON_TRACE_LOGFILE
 #log4j.logger.org.wso2.andes.server.trace=TRACE,CARBON_TRACE_LOGFILE
 
 # Uncomment to enable MessageTracer
-#log4j.logger.org.wso2.andes.tools.utils.MessageTracer=TRACE,CARBON_TRACE_LOGFILE
+log4j.logger.org.wso2.andes.tools.utils.MessageTracer=INFO,CARBON_TRACE_LOGFILE
 
```

Comment:
It seems that the carbon registry which contains log4j mappings reads initial configurations from this file. In this specific case, the appender file is different. Therefore this entry is needed to guide the logger to the right file. 

Will test the if the other classes not mentioned in this file also have the same issue.